tests/UnifiedSpecTests/UnifiedTestRunner.php

```diff
@@ -349,6 +354,48 @@ private function isAuthenticated(): bool
         throw new UnexpectedValueException('Could not determine authentication status');
     }
 
+    /**
+     * Return whether client-side encryption is supported.
+     */
+    private function isClientSideEncryptionSupported(): bool
+    {
+        /* CSFLE technically requires FCV 4.2+ but this is sufficient since we
+         * do not test on mixed-version clusters. */
+        if (version_compare($this->getServerVersion(), '4.2', '<')) {
+            return false;
+        }
+
+        if (FunctionalTestCase::getModuleInfo('libmongocrypt') === 'disabled') {
+            return false;
+        }
```

Comment:
Nit: this could also be extracted to a `isDriverWithMongocrypt` or something similar.

Reply:
Noted. I opted not to do so since it's a one-liner and only called from this context.

I had [`FunctionalTestCase::skipIfClientSideEncryptionIsNotSupported()`](https://github.com/mongodb/mongo-php-library/blob/1.13.0/tests/FunctionalTestCase.php#L509-L518) in mind when writing this.